api/authn/user_store.go

```diff
@@ -0,0 +1,65 @@
+package authn
+
+import (
+	"github.com/pkg/errors"
+	"github.com/rancher/norman/types"
+	"github.com/rancher/types/client/management/v3"
+	"golang.org/x/crypto/bcrypt"
+)
+
+type userStore struct {
+	store types.Store
+}
+
+func SetUserStore(schema *types.Schema) {
+	schema.Store = &userStore{store: schema.Store}
+}
+
+func (s *userStore) ByID(apiContext *types.APIContext, schema *types.Schema, id string) (map[string]interface{}, error) {
+	return s.store.ByID(apiContext, schema, id)
```

Comment:
If you just change the struct to ```go
type userStore struct {
  types.Store
}```

Then you embed the parent type and the delegate methods here ByID and List can be delete.